### ISSUE TITLE
Need to fix reassess button icon on assesment chat page.

### ISSUE THREAD

**Issue by sreejeet** [nic-delhi/AarogyaSetu_Android#7]
The reassess button icon on the top right corner shows up as a box with an X in it.
The logo needs to be fixed as without a clear icon or captions and it's purpose is ambiguous.

**Comment by Animesh-Ghosh** [comment]
Same issue occurs on my device.

Device: Vivo Y66
Android version: 6

**Comment by sreejeet** [comment]
A summary of above mentioned duplicate issues:

1. Pressing the back button tends to close the app and not the currently viewing popup like FAQ, media, or search filter for labs.
2. Font issues like overflow/partially hidden when using native fonts.

**Comment by aravindvnair99** [comment]
# Summary so far:

(From #7 by @sreejeet)

1. The reassess/reset button icon on the top right corner of the self-assessment page shows up as a green box with an X in it. [(Reference)](https://user-images.githubusercontent.com/59018927/82975583-8463e100-9ffa-11ea-9f55-62d77223657b.jpg)

(From #79 by @nihalkza)

2. Pressing the back button tends to close the app and not the currently viewing popup like FAQ, media, search filter for labs, etc.
3. Text overflows or is partially hidden when using native fonts in various places such as buttons and text view. [(Reference)](https://user-images.githubusercontent.com/37146364/82985124-17a81100-a011-11ea-9a92-278b54678347.jpeg)

(From #137 by @ArjitRout)

4. Overflow in split-screen / multitasking / freeform mode. [(Reference)](https://user-images.githubusercontent.com/37146364/82985124-17a81100-a011-11ea-9a92-278b54678347.jpeg)

(From #296 by @nihalkza)

5. Swiping left or right gesture to easily navigate between (Your Status), (Media),(COVID Updates) and (E-pass)

**Comment by aravindvnair99** [comment]
@sreejeet How about keeping this issue tracker for all minor UI/UX issues or improvements. I have consolidated the above list after going through all issues posted so far. You can update the title to `UI/UX issues or improvements` and we can keep track here to avoid people from duplicating so many times (18 times already to be precise).

**Comment by sreejeet** [comment]
@aravindvnair99 The scope of "UI/UX issues or improvements" would be too broad for a single issue tracker and is generally not a good idea.

I prefer flagging only those that come under fonts / icons / back-button action issues. That should keep the scope small, yet help reduce duplicates.

Also, thank you for the summary above.

**Comment by aravindvnair99** [comment]
> @aravindvnair99 The scope of "UI/UX issues or improvements" would be too broad for a single issue tracker and is generally not a good idea.
> 
> I prefer flagging only those that come under fonts / icons / back-button action issues. That should keep the scope small, yet help reduce duplicates.
> 
> Also, thank you for the summary above.

@sreejeet I agree. The 5 I have listed correspond to the title in that case. Shall only flag similar ones henceforth.
